### PR TITLE
Fix grammar

### PR DESCRIPTION
**Pull request by mariumfirdous**
### Description

What component does this PR affect? 

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

Requires changes from other components? If yes, please mark the components:

- [ ] Server
- [ ] Console
- [ ] CLI
- [x] Docs
- [ ] Community Content
- [ ] Build System

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs update
- [x] Community content

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/hasura/graphql-engine/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation. 
- [ ] I have updated the documentation accordingly.
- [ ] I have added required tests.

### CODE REVIEW

**Comment by mariumfirdous** [comment]
#792

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ worst that can happen is that you'll be politely asked to change something. We
 appreciate any sort of contributions and don't want a wall of rules to get in
 the way of that.
 
-However, for those individuals who want a bit more guidance on the best way to
-contribute to the project, read on. This document will cover what we're looking
-for. By addressing all the points we're looking for, it raises the chances we
+However, for those who want a bit more guidance on the best way to
+contribute to the project, read on. This document will cover what we are looking
```
Comment (dsandip) [inline CONTRIBUTING.md]: Contractions are supposed to be better for readability. Let's let them be.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ worst that can happen is that you'll be politely asked to change something. We
...
+for. By addressing all the points we are looking for, it raises the chances we
```
Comment (dsandip) [inline CONTRIBUTING.md]: Contractions are supposed to be better for readability.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -56,10 +56,10 @@ before-mentioned sections).
 
 ## Common guidelines
 
-- Please make sure there is an issue associated with the work that you're doing.
+- Please make sure there is an issue associated with the work that you are doing.
```
Comment (dsandip) [inline CONTRIBUTING.md]: Contractions are supposed to be better for readability.

File: README.md [hunk 4]
```diff
@@ -123,7 +123,7 @@ Check out all the example applications in the
   - [Browse APIs](https://hasura-realtime-group-chat.herokuapp.com/)
 
 - Live location tracking app that shows a running vehicle changing current GPS
```
Comment (dsandip) [inline README.md]:
```suggestion
- Live location tracking app that shows a running vehicle changing its current position
```

File: README.md [hunk 5]
```diff
@@ -123,7 +123,7 @@ Check out all the example applications in the
...
-  coordinates moving on a map. 
+  coordinates moving on the map.
```
Comment (dsandip) [inline README.md]:
```suggestion
  on a map. 
```

File: README.md [hunk 6]
```diff
@@ -162,7 +162,7 @@ Check out our [contributing guide](CONTRIBUTING.md) for more details.
 
 Hasura brand assets (logos, the Hasura mascot, powered by badges etc.) can be
 found in the [assets/brand](assets/brand) folder. Feel free to use them in your
-application/website etc. We'd be thrilled if you add the "Powered by Hasura"
+application/website etc. We would be thrilled if you will add the "Powered by Hasura"
```
Comment (dsandip) [inline README.md]:
```suggestion
application/website etc. We'd be thrilled if you added the "Powered by Hasura"
```

**Comment by dsandip** [comment]
@mariumfirdous I am going to close this PR. Please drop a note here once you've pushed more commits, fixing the requested changes.

**Comment by Soundug** [comment]
@dsandip Can I take up this one and update as per the changes suggested by you?

**Pull request closed without merging**